<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homepage</class>
 <widget class="QWidget" name="homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>831</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout1" stretch="2,2">
    <item>
     <widget class="QLabel" name="admin_op">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>管理员操作</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout1_1">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="one_book_in_btn">
        <property name="text">
         <string>图书单本入库</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="card_manage_btn">
        <property name="text">
         <string>借书证管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="all_book_in_btn">
        <property name="text">
         <string>图书批量入库</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>831</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout2">
    <item>
     <widget class="QLabel" name="user_op">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户操作</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout2_2">
      <property name="spacing">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="book_search_btn">
        <property name="text">
         <string>图书查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lend_ben">
        <property name="text">
         <string>借书</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="return_btn">
        <property name="text">
         <string>还书</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout2_1">
      <property name="spacing">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="log_search_btn">
        <property name="text">
         <string>借书信息查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="account_query_btn">
        <property name="text">
         <string>账户查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lose_btn">
        <property name="text">
         <string>挂失</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxlabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图书管理系统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
